### ISSUE TITLE
No issue! Just to thank for this ACTUALLY Wonderful APPLICATION MY Man!

### ISSUE THREAD

**Issue by jugovic** [DavidXanatos/priv10#13]
No issue! Just to thank for this ACTUALLY Wonderful APPLICATION MY Man!

U ROCK1

**Comment by YenLegion** [comment]
Agreed! Keep up the good work @DavidXanatos! Very happy with the app, liking it much better than Windows Firewall Control.

**Comment by kekukui** [comment]
I agree, this is some really efficient code, with some great security features that Windows desperately needs.  And it uses much less system resources than other firewalls.  But after using Little Snitch on Mac, I feel so disappointed by everything else:

I have tested ~50 Windows firewalls and I think they all suck.  Some are just gigantic, bloated resource hogs.  Some will corrupt your registry and break things.  They will not revert all the changes they made to your registry & network settings when you uninstall, leaving your system in an unstable condition.  And the ones that work are too buggy or the user interface is too complicated.  But Little Snitch is practically perfect: its so much more clean & intuitive than anything else -- especially alerts & rule management.  I think you should copy that user interface as closely as possible.

https://www.obdev.at/products/littlesnitch/getting-started.html

![688_1](https://user-images.githubusercontent.com/20834957/71170981-ad51a000-222a-11ea-97c1-030a82e64992.jpg)

![littlesnitchsimplebbeditalert-100014741-orig 1](https://user-images.githubusercontent.com/20834957/71171683-5ea50580-222c-11ea-9ab4-b2220f994ca0.png)

![little-snitch](https://user-images.githubusercontent.com/20834957/71170993-b2aeea80-222a-11ea-9400-4de001e06e9f.png)

**Comment by DavidXanatos** [comment]
I don't have a mac neider do I want one, hence I will need some detailed information on how the UI is supposed to behave.

Are there any video's showing it in action or would you make one and explain all the UI aspects you find so usefull.

**Comment by kekukui** [comment]
> **I will need some detailed information on how the UI is supposed to behave.**

Sure, no problem.

Little Snitch 3 The Best macOS Firewall
https://www.youtube.com/watch?v=M7Y5CtFK5Us

Little Snitch 3 overview
https://www.youtube.com/watch?v=o74QGbh_G3U

Firewall for Mac
https://www.youtube.com/watch?v=59rrkUFpB0k

> **explain the UI aspects you find so useful**

Mostly the connection alert dialog & list of network application rules.

> **I don't have a mac neider do I want one,**

Yes, I agree -- I dont want one either.  But the user interface existed before Mac, as NextStep.  Since you have good developer skills, maybe you might want to work on a project like this?

Run Mac apps on Linux
https://www.darlinghq.org/

Run Mac apps on BSD
https://web.archive.org/web/20061205023852/http://kerneltrap.org/node/543

**Comment by CHEF-KOCH** [comment]
I agree the best FOSS firewall for Win 10, outstanding. 👍

**Comment by Coldblackice** [comment]
I agree with everyone above, this is a FANTASTIC piece of software. Very well done, and thanks for sharing. I also echo kekukui's comment above, the UI could use a bit of revising/polishing, but that's secondary to the amazing functionality of this tool.

I love all the data it can track and report on, down to individual histories on a per-program/per-service level. The ability to intercept DNS (and with reverse lookup functionality) is also very useful. I also appreciate your other highly useful tools, like Wumgr and Sandboxie Plus. Thanks David!

**Comment by SomeRandomCats** [comment]
I also think you have done a really nice job so far, and it is nice of you to provide it freely for personal use.

There are indeed some nice GUI aspects of LittleSnitch that it would be wonderful to see you implement, such as others have said, such as the Network Monitor.
